### DISCUSSION TITLE
How to work on platform-specific features?

### DISCUSSION THREAD

**Discussion by v1b3m** [supabase/supabase#7238] [Questions]
I can see that some features require the `IS_PLATFORM` variable to be true(this value is obtained from the env var `NEXT_PUBLIC_IS_PLATFORM`). This value is set to false during local development, but say I wanted to work on a feature that requires this value to be `true`, like the templates. How do I go about that?

**Comment by silentworks** [comment]
Can you give some pointers to what you are referring to here please?
